robot/web/stream/server.go

```diff
@@ -507,11 +507,16 @@ func (server *Server) AddNewStreams(ctx context.Context) error {
 			server.logger.Warn("video streaming not supported on Windows yet")
 			break
 		}
+		framerate, err := server.getFramerateFromCamera(name)
+		if err != nil {
+			server.logger.Debugf("error getting framerate from camera %q: %v", name, err)
+		}
 		// We walk the updated set of `videoSources` and ensure all of the sources are "created" and
 		// "started".
 		config := gostream.StreamConfig{
 			Name:                name,
 			VideoEncoderFactory: server.streamConfig.VideoEncoderFactory,
+			TargetFrameRate:     framerate,
```

Comment:
What happens if this is 0? Can we add a test?

Reply:
> encoder pipeline will default to 30fps in bitrate calculation (the same as the key-frame interval) if it cant pick up framerate.

Will see about adding a test for this.